cumulus/tasks/queue-granules/package.json

```diff
@@ -39,6 +39,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@cumulus/common": "^1.0.1",
+    "@cumulus/cumulus-message-adapter-js": "0.0.1-beta.3",
```

Comment:
We should make this 1.0.0

Reply:
I created CUMULUS-367 to deal with this.